compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -329,6 +329,27 @@ matchIteratorTypes(const llvm::SmallBitVector &rootOuterParallelLoop,
   return true;
 }
 
+/// Method to check if the op with have compatible indexing map on
+/// outer-parallel loops. Currently it means the map needs to be identity on the
+/// those dimensions, ignoring its reduction dimensions.
+static bool hasCompatibleOuterParallelLoops(
+    TilingInterface tileOp, AffineMap indexingMap,
+    const llvm::SmallBitVector &rootOuterParallelLoops) {
+  if (!indexingMap.isProjectedPermutation())
+    return false;
+
+  llvm::SmallBitVector parallelLoops = getOuterParallelLoops(tileOp);
+  if (!matchIteratorTypes(rootOuterParallelLoops, parallelLoops))
+    return false;
+
+  /// Project out the non-parallel dimensions.
```

Comment:
comments in a function should use `//`, not `///`

Reply:
Done.

Reply:
This is not resolved?